packages/integration-test/package.json

```diff
@@ -53,7 +53,7 @@
     "@requestnetwork/transaction-manager": "0.26.9",
     "@requestnetwork/types": "0.35.0",
     "@requestnetwork/utils": "0.35.0",
-    "@truffle/hdwallet-provider": "1.2.3",
+    "@truffle/hdwallet-provider": "2.1.1",
```

Comment:
Please revert the hdwallet-provider dependency change if it's not required for this PR

Reply:
will do